config/clusters/2i2c/neurohackademy.values.yaml

```diff
@@ -0,0 +1,101 @@
+dex:
+  enabled: true
+  hubHostName: neurohackademy.2i2c.cloud
+
+staticWebsite:
+  enabled: true
+  source:
+    git:
+      repo: https://github.com/neurohackademy/zoom2022
+      branch: master
+  ingress:
+    host: neurohackademy.2i2c.cloud
+    path: /zoom
+  githubAuth:
+    enabled: true
```

Comment:
@arokem Do you still wish to have a static site displaying zoom information? And is the repo still the appropriate one?

If yes to the first question, I will need you to complete [these steps](https://infrastructure.2i2c.org/howto/features/static-sites/#grant-access-to-the-private-repo) from the app URL here: https://github.com/apps/neurohackademy-static-site-auth

Reply:
> Do you still wish to have a static site displaying zoom information?

Yes, please.

> And is the repo still the appropriate one?

No, let's use: https://github.com/NeuroHackademy-2023/zoom/

I'll proceed to attempt to complete the steps you indicated.

Reply:
@sgibson91 The instructions you sent say: 

```
Go to the ‘Public page’ of the GitHub app created. This usually is of the form https://github.com/apps/<name-of-app>. You can find this in the information page of the app after you create it, under ‘Public link’
```
What should I put into `<name-of-app>`?

Reply:
It's this URL I linked :) https://github.com/apps/neurohackademy-static-site-auth

Reply:
🤦

OK - done!

Reply:
> No, let's use: [NeuroHackademy-2023/zoom](https://github.com/NeuroHackademy-2023/zoom/)

I can't see the repo, can you confirm the default branch is called `master` and not `main` or something else please?

Reply:
Trying to deploy this, I got the following error, which resolves itself if I switch the branch to main

```
> k logs -c content-clone static-sites-68c6bb8df9-fhwdm
Cloning into '/srv/content/repo'...
warning: Could not find remote branch master to clone.
fatal: Remote branch master not found in upstream origin
```

Reply:
Yes, the default branch is `main`